### DISCUSSION TITLE
Text appears as white rectangles on my custom vulkan renderer

### DISCUSSION THREAD

**Discussion by eladmaimoni** [ocornut/imgui#5491] [New Users Build/Link/Run issues ONLY!]
I am following [Sascha Willems example](https://github.com/SaschaWillems/Vulkan/tree/master/examples/imgui) on how to integrate imgui with a Vulkan renderer.

Trying to render this simple window:

        ImGui::NewFrame();
        ImGui::SetNextWindowSize(ImVec2(128, 64), ImGuiCond_FirstUseEver);
        ImGui::Begin("Test Window", nullptr);
        ImGui::Text("Test Text");
        ImGui::End();
        ImGui::Render();
        
Yields the following:
 
![image](https://user-images.githubusercontent.com/17446835/180049479-1e9d93f4-585d-4779-afa9-5e3ab496c07e.png)

Unfortunately text appears as white rectangles.
I examined the rendering process against Sascha Willem code using Renderdoc but haven't yet found what went wrong.

[Here is a link to the renderdoc file and relevant code files](https://1drv.ms/u/s!ApkpQgJnXKOBicVt8xH_YSdhZeiwGw?e=yNEwvy).

From my examination, it seems that vertex data and textures are loaded correctly. but the two samples have a slightly different font texture and as a result, slightly different uv coordinates per vertex. But nothing to explain why the textured text quads appear completely white. I guess the difference in the font texture emanate from the fact that I use a different imgui version (latest docking branch from vcpkg).

Does anyone have an idea what could lead to such result?

I would be happy to share more info if you think it is relevant (code, renderdoc files) but I wish to avoid clutter this post with too much code.

**Comment by jeremyong** [comment]
Without looking at the code just yet (on mobile) I'm assuming you don't have the blend state set up correctly for alpha blending on your ImGui pso
